chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/DeveloperManagerImpl.java

```diff
@@ -362,6 +390,71 @@ public DeveloperDTO merge(List<Long> developerIdsToMerge, DeveloperDTO developer
         return createdDeveloper;
     }
 
+    private String getDuplicateChplProductNumberErrorMessage(
+            final List<DuplicateChplProdNumber> duplicateChplProdNumbers) {
+
+        String message = "";
+
+        for (DuplicateChplProdNumber dup : duplicateChplProdNumbers) {
+            message += "\n" + dup.toString();
```

Comment:
Needs to use system-specific line separator. I think `\n` is for Linux and `\r\n` is for Windows, but there's some Java property to handles it. That said, I'm not sure this is a good idea anyway, as I don't know if it'll display either of those things correctly in all browsers. There you'd need `<br>`, but that doesn't make sense in an API response at all. Might make more sense to put multiple error messages in the response; one for each pair of duplicates. The UI already handles that kind of thing correctly